README.rst

```diff
@@ -1741,6 +1741,130 @@ If the request is made without passing the ``time`` argument, the server local t
 
 The ``time`` parameter was added to support `resilient collection and sending of data by the OpenWISP Monitoring Agent <https://github.com/openwisp/openwrt-openwisp-monitoring#collecting-vs-sending>`_.
 
+List wifi session
+#################
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/
+
+**Available filters**
+
+You can filter using ``device_id`` to get list of wifi sessions related to that particular device.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?device={device_id}
+
+You can filter using ``organization_id`` to get list of wifi sessions belongs to an organization.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?device__organization={organization_id}
+
+You can filter using ``group_id`` to get list of wifi sessions belongs to a group.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?device__group={group_id}
+
+You can filter to get list of wifi sessions using ``start_time``.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?start_time={start_time}
+
+You can filter to get list of wifi sessions using ``stop_time``.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?stop_time={stop_time}
```

Comment:
Please double check that it is possible to use comparison (`gt`, `gte`, `lt`, `lte`) for filtering with time.

Reply:
Yes, it is possible :+1: